src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,220 @@
+  //Main Container > Table Header
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    padding: rem(22px) 0 0 $spacing-md;
+  }
+
+  .#{$prefix}--data-table-v2-title {
+    @include type-style('productive-heading-03');
+    color: $text-01;
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    @include type-style('body-short-01');
+    color: $text-02;
+  }
+
+  //-------------------------------------------------
+  //Main Container > Table (Default Styles)
+  //-------------------------------------------------
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+
+    thead {
+      @include type-style('body-short-01');
+      background-color: $ui-03;
+      font-weight: 700;
+    }
+
+    tbody {
+      @include type-style('body-short-01');
+      background-color: $ui-01;
+      width: 100%;
+    }
+
+    tr {
+      border: none;
+      height: rem(48px);
+      width: 100%;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) {
+      background: $hover-field;
+    }
+
+    tbody tr:hover:not(.#{$prefix}--row-disabled) td {
+      color: $text-01;
+      background: $hover-field;
+      border-bottom: 1px solid $hover-field;
+      border-top: 1px solid $hover-field;
+    }
+
+    th,
+    td {
+      padding-left: $spacing-sm;
+      padding-right: $spacing-sm;
+      vertical-align: top;
+      text-align: left;
+      color: $text-02;
+      border-top: 1px solid $ui-01;
+      border-bottom: 1px solid $ui-01;
+    }
+
+    th {
+      color: $text-01;
+      border-top: 1px solid $ui-03;
+      border-bottom: 1px solid $ui-03;
+    }
+
+    th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+      padding-left: $spacing-lg;
+    }
+
+    th.#{$prefix}--table-column-icon:hover,
+    th.#{$prefix}--table-column-menu:hover,
+    th.#{$prefix}--table-expand-v2:hover {
+      background: $ui-03;
+    }
+
+    tbody tr td {
+      background: $ui-01;
+      border-bottom: 1px solid $ui-03;
+      border-top: 1px solid $ui-01;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2) {
+      padding: rem(14px) $spacing-sm;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2):first-of-type {
+      padding-left: $spacing-lg;
+    }
+
+    td:not(.#{$prefix}--table-column-checkbox):not(.#{$prefix}--table-column-menu):not(.#{$prefix}--table-column-icon):not(.#{$prefix}--table-expand-v2):last-of-type {
+      padding-right: $spacing-lg;
+    }
+
+    .#{$prefix}--table-header-label {
+      display: block;
+      padding: rem(14px) 0;
+    }
+
+    //width for checkbox columns
+    .#{$prefix}--table-column-checkbox {
+      width: rem(50px);
```

Comment:
Typically our dimension units will be multiple of 8s, if we are doing 50/10 because of the spec then I think it's a design problem, if it's in code we should leave a note as to why it's not following our mini units or spacing scale.